### DISCUSSION TITLE
Custom Difficulty plugin

### DISCUSSION THREAD

**Discussion by clarg18** [Pryaxis/TShock#1948] [Q&A]
Hello,

Looking through the plugins, I can see a customer difficulty plugin which I would really like, but it looks like the dev deleted his github. Does anyone have a copy of it?

https://tshock.co/xf/index.php?resources/customdifficulty.189/

**Comment by Tygra** [comment]
Hey,

Is this what you're looking for? https://github.com/tanpro260196/CustomDifficulty

**Reply by moisterrific** [reply]
Last commit 3 years ago 😬

**Reply by Tygra** [reply]
Well it's 2,1, so not that far behind:)
